#! /usr/bin/env python

PACKAGE = "robotx_nav"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("aim_angular_kp", double_t, 0, "angular kp ", 0.7, -10, 10.0)
gen.add("aim_angular_ki", double_t, 0, "angular ki ", 0.001, -10, 10.0)
gen.add("aim_angular_kd", double_t, 0, "angular kd ", 0.07, -10, 10.0)

gen.add("aim_angular_velocity_threshold", double_t, 0, "angular velocity threshold", 0.3, 0, 3.14)

exit(gen.generate(PACKAGE, "robotx_nav", "Aim"))
